packages/fether-react/src/utils/blockscout.js

```diff
@@ -0,0 +1,43 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+const baseUrl = chainName => {
+  let baseUrl;
+
+  let chainNameBlockscout = '';
+
+  switch (chainName) {
+    case 'foundation':
+      chainNameBlockscout = 'mainnet';
+      baseUrl = `https://blockscout.com/eth/${chainNameBlockscout}`;
+      break;
+    case 'kovan':
+    case 'ropsten':
+    case 'rinkeby':
+      chainNameBlockscout = chainName;
+      baseUrl = `https://blockscout.com/eth/${chainNameBlockscout}`;
+      break;
+    default:
+      console.error(
+        'Chain name not yet supported. Please notify admin@parity.io'
```

Comment:
Although an email is easier to send for anybody, the user hitting this error must be advanced enough to have launched Fether in the console, so I'd suggest to ask to open a github issue on https://github.com/paritytech/fether/issues/new